misc/glossary.md

```diff
@@ -0,0 +1,156 @@
+
+**Protocol-Owned Vault:** Maker vault that can only be utilized by Maker Governance that uses decentralized collateral to generate Dai.
+
+**RWA Adopted Vault:** Vault with RWA collateral that only pays a fixed part of their stability fee to Maker Core. A Protector SubDAO is responsible for liquidations and other losses before Maker Core.
+
+**RWA Collateral Scope:** Primary Scope that interacts with real world assets that include: cashlike (centralized stable coins & bonds), clean money (renewable energy projects), physical resilient, and miscellaneous (WBTC) assets.
+
+**RWA Services:** External services that require interaction with the physical world in order to obtain assets.
+
+**Scope:** Broad focus area of work within which SubDAOs, Ecosystem Actors and Maker Governance operate.
+
+**Scope Framework:** Set of rules that SubDAOs within that Scope must adhere to. 
+
+**Stability and Liquidity Scope:** Secondary Scope that includes cashlike RWA portfolio management, target rate management, and protocol owned vault management.
+
+**Stability Fee Base Rate (SFBR):** Interest rate which is added to the stability fee rate of some or all vaults.
+
+**Stance:** Set of rules during Endgame that determine the type of collateral backing Dai as well as the value of Dai.
+
+**SubDAOs:** Entity within the Maker Ecosystem that functions as self-sustaining DAOs. SubDAOs maintain a synergistic relationship with Maker Core.
+
+**SubDAO Farms:** Mechanism to distribute SubDAO tokens as rewards to Dai holders, Maker Governance participants, and EtherDai vault users.
+
+**SubDAO Frontends:** Frontends that allow users to interact with a SubDAO. The frontends include several standard functions and design elements.
+
+**SubDAO Tokens:** ERC-20 tokens used to participate in SubDAO governance.
+
+**SubElixir:** Liquidity pool consisting of SubDAO tokens and MKR.
+
+**Supporting Scopes:** Scope Category that includes Protocol Engineering, Growth, Physical Resilence, Infratructure, Interface, and Finance.
+
+**Target Rate (TR):** Goal for the rate at which the target price of Dai changes over time.
+
+**Tribunal:** Entity that is responsible for the creation and adherence to Supporting Scope Frameworks. 
+
+**Unified Coordination Toolkit (UCT):** Open-source software that is utilized as a transparency and governance reporting tool as well as a collaboration tool.
+
+**Unrecognized Delegates:** Delegates that are completely anonymous who do not receive compensation for governance voting.
+
+**Vault Engine:** Regulates the amount of Dai that may be minted by vaults on L2s or vaults operated by SubDAOs.
+
+**Workforce Bonus:** MKR that is utilized to compensate Delegates, Council, and Core Unit members.
+
+**Workforce Bonus Pool:** A pool initially consisting of SubDAO tokens that is distributed based upon SubDAO token governance to incentivize the creation and maintenance of SubDAOs.
+
+
+  
+## Deprecated terms
```

Comment:
Should include stuff like MetaDAO + MetaElixir

Reply:
Potentially Elixir as well, but I haven't quite surrendered to that being renamed yet.